### PR TITLE
Refactor MTE-147 [v113] Reduce credits by using same stage

### PR DESCRIPTION
**Pull request by isabelrios**
Let's try to use the Fennec build for testing to run the UI tests and so reduce one stage and the credits it takes to run it.

### CODE REVIEW

**Comment by isabelrios** [comment]
We can use `configure_fennec` step in the first stage only, with it, we can build for testing to run the next stage which is in charge of the XCUITests. By having just this config step and removing the other configure_fennec_xcuitest, we save around 25 credits / build

In addition, I'm moving one test suite from smoketest 3 to smoketest 2 to have similar test execution times.

File: bitrise.yml [hunk 1]
```diff
@@ -365,18 +305,21 @@ workflows:
 
             echo "$BITRISE_ARTIFACT_PATHS"
 
+            derived_data=${BITRISE_ARTIFACT_PATHS#*|}
```
Comment (AaronMT) [inline bitrise.yml:308]: Not familiar with the `#*|` but I assume its needed.

**Review by AaronMT — APPROVED** [review]
Looks OK to me

File: bitrise.yml (hunk 1 above)
Comment (isabelrios) [reply bitrise.yml:308]: yes, the reason is that in the BITRISE_ARTIFACT_PATHS there are two .zip files, they appear as:
`BITRISE_ARTIFACT_PATHS=/foo/bar/file1.zip|/foo/bar/file2.zip`
We need to unzip only the second one, the | character is the separator between both. With that command we get what is after the pipe.
Not sure if there is a better way, happy to learn and use it if that's the case.

**Pull request merged**